graphql2/schema.graphql

```diff
@@ -1006,3 +1008,15 @@ type AuthSubject {
   subjectID: ID!
   userID: ID!
 }
+
+type Notice {
+  type: NoticeType!
+  message: String!
+  details: String
```

Comment:
I think we should make this required (since it's a non-structural output) -- it can still be empty and thus falsy.
```suggestion
  details: String!
```